config/freebsd/newsyslog/waagent.conf

```diff
@@ -0,0 +1 @@
+/var/log/waagent.log			644  7	   *	@T00  J     /var/run/waagent.pid
```

Comment:
A couple of questions:

- If I'm reading the documentation correctly, by adding waagent.pid, the agent would be sent a SIGHUP on rotation... how does the agent handle this signal.

- From the documentation it was not clear to me how optional fields are specified. I see that you skipped owner (2nd field). Are the TABs between "waagent.log" and "644" significant? I also see a TAB between "*" and "@T00"

Thanks

Reply:
Hi, man page of newsyslog.conf reads as: "Each line has five mandatory fields and four optional fields, separated with whitespace". Any number of TABs or SPACEs count as whitespace.

Reply:
Hi,

SIGHUP is not needed as FileAppender.write in azurelinuxagent/common/logger.py closes the logfile after each write. I removed the pid file from the newsyslog config.

BR, Felix